qiskit/primitives/base_estimator.py

```diff
@@ -0,0 +1,199 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+r"""
+=========
+Estimator
+=========
+
+Estimator class estimates expectation values of quantum circuits and observables.
+
+An estimator object is initialized with multiple quantum circuits and observables
+and users can specify pairs of quantum circuits and observables
+to estimate the expectation values.
+
+The input consists of following elements.
+
+* quantum circuits (:math:`\psi_i(\theta)`): list of (parametrized) quantum circuits
+  or state vectors to be converted into quantum circuits.
+  (a list of :class:`~qiskit.circuit.QuantumCircuit`))
+
+* observables (:math:`H_j`): a list of :class:`~qiskit.quantum_info.SparsePauliOp`.
```

Comment:
also the Aer & quantum_info implementation can maybe allow any Hermitian operator here?

Reply:
Yes, it can be any BaseOperator. SparsePauliOp with complex coefficients (non-Hermitian operator) is also acceptable.